lms/static/sass/views/_login-register.scss

```diff
@@ -224,6 +224,15 @@
         &:focus {
             outline: none;
         }
+
+        div[class^="hidden"] {
```

Comment:
I think the `class` attribute values on those divs look like `form-field hidden-username`. You could use `div[class*="hidden-"]` instead.